dd-trace-api/src/main/java/datadog/trace/api/Config.java

```diff
@@ -155,7 +164,7 @@
         getBooleanSettingFromEnvironment(TRACE_RESOLVER_ENABLED, DEFAULT_TRACE_RESOLVER_ENABLED);
     serviceMapping = getMapSettingFromEnvironment(SERVICE_MAPPING, null);
 
-    globalTags = getMapSettingFromEnvironment(GLOBAL_TAGS, null);
+    globalTags = getMapSettingFromEnvironment(GLOBAL_TAGS, DEFAULT_GLOBAL_TAGS);
```

Comment:
Does this still ad the `DEFAULT_GLOBAL_TAGS`, even if global tags are passed in from the environment/system properties?

Reply:
No, but configuring the global tags wouldn't need to be advertised to users, who should be using `trace.span.tags` with the same functionality as we update the env var.

I could update `Config.getMergedSpanTags()` such that it's permanent if you'd prefer, though this was a bit less invasive and allows the option of overriding if desired.

Reply:
That's fine